### ISSUE TITLE
Validate `accessibilitySupport` values against controlled vocabulary

### ISSUE THREAD

**Issue by nickevansuk** [openactive/data-model-validator#356]
Currently only `activity` values are validated. `accessibilitySupport` values should be validated against https://openactive.io/accessibility-support/.
